packages/types/src/authentication.ts

```diff
@@ -43,7 +43,7 @@ export type SignUpFailureReasons = z.infer<typeof SignUpFailureReasons>;
 
 export const SignUpFailureResponseBody = z.object({
     status: z.literal('FAILURE'),
-    signFailureReasons: SignUpFailureReasons.array(),
+    signUpFailureReasonCollection: SignUpFailureReasons.array(),
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** 👍